specs/README.md

```diff
@@ -0,0 +1,22 @@
+# Celestia App Specifications
+
+## Building From Source
+
+To build book:
```

Comment:
[optional]

```suggestion
Prerequisite: [install mdBook](https://rust-lang.github.io/mdBook/guide/installation.html)

To build book:
```

Reply:
this is a good addition, but I'd prefer not to add it in this PR

Reply:
Fixed in https://github.com/celestiaorg/celestia-app/pull/800